reference/clever-tools/networkgroups.md

```diff
@@ -0,0 +1,192 @@
+```
+
+```log
+Networkgroup 'first-ng' was created with the id 'ng_bcce10f3-ec6a-4d51-8e28-50cd622d2ecb'.
+```
+
+**Tips:**
+
+- You can add `--tags tag1,tag2,tag3` to provide tags for networkgroups management.
+- [Use `--interactive` to answer optional questions](#use---interactive-to-answer-optional-questions)
+
+### List created networkgroups
+
+To check the list of networkgroups you created, you can run:
+
+```sh
+clever ng list
+```
+
+After [Create a networkgroup](#create-a-networkgroup), you should get:
+
+```log
+Networkgroup ID                           Label                 Members  Peers  Description
+────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+ng_bcce10f3-ec6a-4d51-8e28-50cd622d2ecb   first-ng              0        0      My first networkgroup
+```
+
+**Tip:** [`--json` or `-j` shows output as JSON](#--json-or--j-shows-output-as-json)
+
+### Add a member
+
+After [creating a networkgroup](#create-a-networkgroup), you can add members to it. A member is an app, an add-on or an [external node category](TODO: Redirect to doc). You can set `--type` to `'application'`, `'addon'` or `'external'` according to your needs.
+
+For example, to add an app to the networkgroup, you can do:
+
+```sh
+clever ng --ng 'first-ng' members add --member-id 'app_d60ef23d-80d7-4095-b8d7-7f42bea137ea' --type 'application' --domain-name 'my-app-1' --label 'My app 1'
+```
+
+```log
+Successfully added member 'app_d60ef23d-80d7-4095-b8d7-7f42bea137ea' to networkgroup 'ng_bcce10f3-ec6a-4d51-8e28-50cd622d2ecb'.
+```
+
+**Tips:**
+
+- [`--ng` takes a networkgroup label or ID](#--ng-takes-a-networkgroup-label-or-id)
+- `--label` is optional, but it's a good practice to add it.
+
+### List members in a networkgroup
+
+To list the members in a networkgroup, run `clever ng members list`. For example, after ["Add a member"](#add-a-member), you could run:
+
+```sh
+clever ng --ng 'first-ng' members list
+```
+
+You should get something like:
+
+```log
+Member ID                                 Member Type                Label                                     Domain Name
+───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+app_d60ef23d-80d7-4095-b8d7-7f42bea137ea  application                My app 1                                  my-app-1
+```
+
+**Tips:**
+
+- [`--ng` takes a networkgroup label or ID](#--ng-takes-a-networkgroup-label-or-id)
+- [`--json` or `-j` shows output as JSON](#--json-or--j-shows-output-as-json)
+
+### Add an external node category
+
+External node categories are <TODO>. You can have plenty of them, as it helps keeping this organized.
+
+Adding an external node category is very similar to the example in ["Add a member"](#add-a-member), except you'll use a custom member ID and `'external'` for the `--type`.
+
+Here is an example:
```

Comment:
Those example-introducing lines are a bit useless since the example commands speak for themselves.